<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ivo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/icons/icons/stupid_logo.png</normaloff>:/icons/icons/stupid_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenuBar {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(234, 234, 234, 255), stop:1 rgba(255, 255, 255, 255));
}

QToolBar {
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(234, 234, 234, 255));
border-color: transparent;
}

QDockWidget::title {
border: 1px solid lightgrey;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}

QToolButton {
border-radius: 4px;
border: 1px solid  rgba(0, 0, 0, 0);
}

QToolButton::hover {
border-radius: 4px;
border: 1px solid  rgba(220, 220, 255, 120);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 10), stop:1 rgba(190, 210, 255, 255));
}

QToolButton::checked {
border-radius: 4px;
border: 1px solid  rgba(210, 210, 230, 120);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 35), stop:1 rgba(240, 210, 160, 255));
}</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_Model"/>
    <addaction name="actionOpen_obj"/>
    <addaction name="actionLoad_Texture"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave"/>
    <addaction name="actionCloseModel"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Sheets"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionToggle_Lighting"/>
    <addaction name="actionShow_Grid"/>
    <addaction name="actionZoom_fit"/>
    <addaction name="actionZoom_2D"/>
    <addaction name="actionZoom_3D"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionAutoPack"/>
    <addaction name="actionScale"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Model"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_obj"/>
   <addaction name="actionLoad_Texture"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionModeSelect"/>
   <addaction name="actionModeMove"/>
   <addaction name="actionModeRotate"/>
   <addaction name="actionModeSnap"/>
   <addaction name="actionModeFlaps"/>
   <addaction name="separator"/>
   <addaction name="actionScale"/>
   <addaction name="actionPolypaint"/>
   <addaction name="actionAutoPack"/>
   <addaction name="separator"/>
   <addaction name="actionExport_Sheets"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget3D">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>3D Model</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="CRenWin3D" name="frameLeft"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget2D">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>2D Layout</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="CRenWin2D" name="frameRight"/>
  </widget>
  <action name="actionOpen_obj">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-new.png</normaloff>:/icons/icons/48px-Document-new.png</iconset>
   </property>
   <property name="text">
    <string>Import &amp;Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-save-as.png</normaloff>:/icons/icons/48px-Document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-save.png</normaloff>:/icons/icons/48px-Document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-System-log-out.png</normaloff>:/icons/icons/48px-System-log-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionLoad_Texture">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Image-x-generic.png</normaloff>:/icons/icons/48px-Image-x-generic.png</iconset>
   </property>
   <property name="text">
    <string>Mate&amp;rial Manager</string>
   </property>
   <property name="toolTip">
    <string>Browse current model's materials and assign textures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Help-browser.png</normaloff>:/icons/icons/Help-browser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionModeRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-refresh.png</normaloff>:/icons/icons/48px-View-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionModeSnap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Go-bottom.png</normaloff>:/icons/icons/48px-Go-bottom.png</iconset>
   </property>
   <property name="text">
    <string>Join/Break edges</string>
   </property>
   <property name="toolTip">
    <string>Join/Break edges</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionModeMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Move_mode.png</normaloff>:/icons/icons/Move_mode.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionModeFlaps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Flaps_mode.png</normaloff>:/icons/icons/Flaps_mode.png</iconset>
   </property>
   <property name="text">
    <string>Switch flaps</string>
   </property>
   <property name="toolTip">
    <string>Switch flaps</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionExport_Sheets">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Printer.png</normaloff>:/icons/icons/48px-Printer.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xport Sheets</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Preferences-system.png</normaloff>:/icons/icons/48px-Preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Sett&amp;ings</string>
   </property>
   <property name="toolTip">
    <string>Edit rendering, paper and export settings</string>
   </property>
  </action>
  <action name="actionZoom_fit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-zoom-fit.png</normaloff>:/icons/icons/48px-View-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom fit windows</string>
   </property>
  </action>
  <action name="actionZoom_2D">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-zoom-fit.png</normaloff>:/icons/icons/48px-View-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;2D</string>
   </property>
   <property name="toolTip">
    <string>Zoom fit 2D window</string>
   </property>
  </action>
  <action name="actionZoom_3D">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-View-zoom-fit.png</normaloff>:/icons/icons/48px-View-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;3D</string>
   </property>
   <property name="toolTip">
    <string>Zoom fit 3D window</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Edit-undo.png</normaloff>:/icons/icons/48px-Edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Edit-redo.png</normaloff>:/icons/icons/48px-Edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionLoad_Model">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Document-open.png</normaloff>:/icons/icons/48px-Document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Scale.png</normaloff>:/icons/icons/Scale.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Scale</string>
   </property>
  </action>
  <action name="actionAutoPack">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Package-x-generic.svg.png</normaloff>:/icons/icons/48px-Package-x-generic.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Auto Pack</string>
   </property>
   <property name="toolTip">
    <string>Automatically arrange parts in 2D window</string>
   </property>
  </action>
  <action name="actionShow_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Antialias_Icon.svg.png</normaloff>:/icons/icons/48px-Antialias_Icon.svg.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Grid</string>
   </property>
   <property name="toolTip">
    <string>Show grid in 3D view window</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionToggle_Lighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/light_bulb.png</normaloff>:/icons/icons/light_bulb.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Toggle Lighting</string>
   </property>
   <property name="toolTip">
    <string>Toggle Lighting</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionPolypaint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/48px-Polypaint.png</normaloff>:/icons/icons/48px-Polypaint.png</iconset>
   </property>
   <property name="text">
    <string>Polypaint</string>
   </property>
   <property name="toolTip">
    <string>Enter Polypaint mode</string>
   </property>
  </action>
  <action name="actionModeSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/Select_mode.png</normaloff>:/icons/icons/Select_mode.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>`</string>
   </property>
  </action>
  <action name="actionCloseModel">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/icons/icons/close.png</normaloff>:/icons/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Model</string>
   </property>
   <property name="toolTip">
    <string>Close currently opened model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CRenWin3D</class>
   <extends>QWidget</extends>
   <header>interface/renwin3d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CRenWin2D</class>
   <extends>QWidget</extends>
   <header>interface/renwin2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
